actions/changelog-generate/action.yaml

```diff
@@ -0,0 +1,148 @@
+name: "Generate changelog"
+description: "Generate changelog after release"
+inputs:
+  # required inputs
+  changelog-file:
+    description: Path to the Changelog.md file in the github repository
+    required: false
+    default: "CHANGELOG.md"
+  github-token:
+    description: "The GitHub token for committing the changes."
+    required: true
+  new-tag:
+    description: Version after bump
+    required: true
+  old-tag:
+    description: Previous version. Let empty for releases
+    required: false
+    default: ""
+
+  # unrequired inputs
+  config:
+    description: Path to the changelog config JSON file
+    required: false
+    default: "./.github/changelog-config.json"
+  output:
+    description: "Name of the output file for the generate changelog step"
+    required: false
+    default: "changes.md"
+  fetchReviewers:
+    description: "Will enable fetching the users/reviewers who approved the PR"
+    required: false
+    default: "false"
+  fetchReleaseInformation:
+    description: "Will enable fetching additional release information from tags."
+    required: false
+    default: "false"
+  commitMode:
+    description: "Special configuration for projects which work without PRs."
+    required: false
+    default: "false"
```

Comment:
@yannick-roeder  The used composite action allows it. so i wanted to include here. and since the default is set to false we should not have to deal with it if not needed